### ISSUE TITLE
Compatiblity Issue with Spree 3.7 and 4

### ISSUE THREAD

**Issue by 4goodapp** [bricesanchez/spree-refinery-authentication#17]
When I add this extension to a Spree version 3.7 app an run the command `bundle`, I get this: 

```
➜ bundle 
Fetching https://github.com/refinerycms-contrib/spree-refinery-authentication.git
Fetching gem metadata from https://rubygems.org/..........
Fetching gem metadata from https://rubygems.org/.
Resolving dependencies...
Bundler could not find compatible versions for gem "deface":
  In snapshot (Gemfile.lock):
    deface (= 1.5.3)

  In Gemfile:
    spree-refinerycms-authentication was resolved to 3.0.0, which depends on
      deface (~> 1.0.0)

    spree_gateway (~> 3.4) was resolved to 3.6.7, which depends on
      spree_core (>= 3.1.0, < 5.0) was resolved to 3.7.10, which depends on
        deface (~> 1.0)

Running `bundle update` will rebuild your snapshot from scratch, using only
the gems in your Gemfile, which may resolve the conflict.
```
How can I solve this Issue?

Edit:
This is what I added in my Gemfile looks like:
```
gem 'refinerycms', '~> 3.0.0'
gem 'spree-refinerycms-authentication', github: 'refinerycms-contrib/spree-refinery-authentication', branch: '3-0-stable'

```

Rails version: 5.2.4.1
Ruby: 2.5.1

**Comment by bricesanchez** [comment]
Hi @4goodapp !

Thanks for using this extension.

I've updated the `3-0-stable` branch with the latest changes on `master` branch, this should fix your issue.

**Comment by 4goodapp** [comment]
Hi, @bricesanchez 

When I run `bundle install`, I get this error:

```
➜ bundle install
Fetching https://github.com/refinery/refinerycms.git
Fetching gem metadata from https://rubygems.org/..........
Fetching gem metadata from https://rubygems.org/.
Resolving dependencies...
Bundler could not find compatible versions for gem "railties":
  In snapshot (Gemfile.lock):
    railties (= 5.2.4.1)

  In Gemfile:
    spree_auth_devise (~> 3.5) was resolved to 3.5.2, which depends on
      devise-encryptable (= 0.2.0) was resolved to 0.2.0, which depends on
        devise (>= 2.1.0) was resolved to 4.6.2, which depends on
          railties (>= 4.1.0, < 6.0)

    refinerycms was resolved to 4.1.0, which depends on
      refinerycms-core (= 4.1.0) was resolved to 4.1.0, which depends on
        railties (>= 6.0.0, < 7)

Running `bundle update` will rebuild your snapshot from scratch, using only
the gems in your Gemfile, which may resolve the conflict.
```

On Spree 3.7.
